.eslintrc.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+    env: {
+        browser: true,
+        es2020: true
+    },
+    extends: 'standard',
+    globals: {
+        Thenable: 'readonly'
+    },
+    parser: '@typescript-eslint/parser',
+    parserOptions: {
+        ecmaVersion: '2020',
+        sourceType: 'module'
+    },
+    plugins: [
+        '@typescript-eslint',
+        'import',
+        'unused-imports'
+    ],
+    rules: {
+        indent: ['error', 4, {
+            SwitchCase: 1
+        }],
+        'space-before-function-paren': ['error', {
+            anonymous: 'always',
+            named: 'never',
+            asyncArrow: 'always'
+        }],
+        semi: ['error', 'always'],
+        'no-extra-semi': 'off',
+        '@typescript-eslint/no-extra-semi': ['error'],
+        'no-dupe-class-members': 'off',
```

Comment:
That's the base javascript rule that doesn't work well with typescript, what about enabling `@typescript-eslint/no-dupe-class-members`?